src/dotnet/devcontainer-feature.json

```diff
@@ -1,35 +1,33 @@
 {
     "id": "dotnet",
-    "version": "1.1.4",
+    "version": "2.0.0",
     "name": "Dotnet CLI",
     "documentationURL": "https://github.com/devcontainers/features/tree/main/src/dotnet",
-    "description": "Installs the .NET CLI. Provides option of installing sdk or runtime, and option of versions to install. Uses latest version of .NET sdk as defaults to install.",
+    "description": "Installs .NET SDK, which includes the .NET CLI and the shared runtime. Offers options to install multiple SDK versions.",
     "options": {
         "version": {
             "type": "string",
             "proposals": [
                 "latest",
-                "7",
-                "6",
-                "3.1"
+                "lts",
+                "8.0",
+                "7.0",
+                "6.0"
             ],
             "default": "latest",
-            "description": "Select or enter a dotnet CLI version. (Available versions may vary by Linux distribution.)"
+            "description": "Version of .NET SDK to install. Use 'latest' for the latest version, 'lts' for the latest LTS version, 'X.Y' or 'X.Y.Z' for a specific version."
         },
-        "runtimeOnly": {
-            "type": "boolean",
-            "default": false,
-            "description": "Install just the dotnet runtime if true, and sdk if false."
-        },
-        "installUsingApt": {
-            "type": "boolean",
-            "default": true,
-            "description": "If true, it installs using apt instead of the release URL"
+        "additionalVersions": {
+            "type": "string",
+            "default": "",
+            "description": "Comma-separated list of additional .NET SDK versions to install. Use 'latest' for the latest version, 'lts' for the latest LTS version, 'X.Y' or 'X.Y.Z' for a specific version."
         }
     },
     "containerEnv": {
-        "DOTNET_ROOT": "/usr/local/dotnet/current",
-        "PATH": "/usr/local/dotnet/current:${PATH}"
+        "DOTNET_ROOT": "/usr/share/dotnet",
+        "PATH": "$PATH:$DOTNET_ROOT:~/.dotnet/tools",
```

Comment:
> ~/.dotnet/tools

From https://github.com/devcontainers/cli/issues/391, I don't expect ^ to work? 🤔

Reply:
Hmm 🙃, what did you mean by "that doesn't work if a shell is not invoked."?

I wrote the following test to make sure running a tool from this directory works in bash.

https://github.com/devcontainers/features/blob/3d48f7749e8e74477d26e23dc9a6e4f4f8ae77e3/test/dotnet/install_dotnet_global_tool.sh#L16-L25

Reply:
<img width="768" alt="Cannot find executable in dotnet/tools" src="https://github.com/devcontainers/features/assets/19893438/9236c9d3-57e1-4e1b-b96f-7fdf85668d9c">

This doesn't seem to work in a codespace. Link to create one: https://github.com/codespaces/new/?skip_quickstart=true&repo=449816349&ref=sliekens-dotnet&devcontainer_path=.devcontainer%2Fdevcontainer.json

Reply:
I'm not 100% sure why `which` doesn't see it, but...

![image](https://github.com/devcontainers/features/assets/1583241/0da4d817-9dae-4b56-860f-c02c38082dec)

Reply:
Another example

![image](https://github.com/devcontainers/features/assets/1583241/b78d81e6-8872-496c-872e-cdcc6731716d)

Reply:
(Marking resolved because I think this works as intended)